.github/workflows/external.yml

```diff
@@ -0,0 +1,27 @@
+name: Update dependencies
+on:
+  workflow_dispatch: {}
+  # schedule:
+  #   - cron: '34 3 * * */8'
+
+jobs:
+  check:
```

Comment:
I'd use https://github.com/actions/setup-python and have some `requirements.txt`. Not sure if all packages used in the script are expected to be installed.

Reply:
All packages imported are included in python3 by default, and the GitHub Actions runner already has python3 set up I believe.